bin/start-coordinator.sh

```diff
@@ -28,21 +28,39 @@ COORDINATOR_HOME="$(
 CONF_FILE="./conf/coordinator.conf "
 MAIN_CLASS="com.tencent.rss.coordinator.CoordinatorServer"
 
+cd $COORDINATOR_HOME
+
 source "${COORDINATOR_HOME}/bin/rss-env.sh"
 source "${COORDINATOR_HOME}/bin/utils.sh"
 
+HADOOP_CONF_DIR=$HADOOP_HOME/etc/hadoop
```

Comment:
why coordinator should depend on Hadoop jars?

Reply:
The access checker needs to be able to read candidates file in hdfs